holidays/countries/__init__.py

```
@@ -12,191 +12,191 @@
 )
-from .united_states_virgin_islands import UnitedStatesVirginIslands, VI, VIR, HolidaysVI
-from .uruguay import Uruguay, UY, URY
-from .uzbekistan import Uzbekistan, UZ, UZB
-from .vanuatu import Vanuatu, VU, VTU
-from .vatican_city import VaticanCity, VA, VAT
-from .venezuela import Venezuela, VE, VEN
-from .vietnam import Vietnam, VN, VNM
-from .yemen import Yemen, YE, YEM
-from .zambia import Zambia, ZM, ZMB
-from .zimbabwe import Zimbabwe, ZW, ZWE
+from holidays.countries.united_states_virgin_islands import UnitedStatesVirginIslands, VI, VIR, HolidaysVI
```

Ruff (0.11.9)

Line too long (106 > 99)

(E501)